spec/ttt/board_spec.clj

```diff
@@ -0,0 +1,19 @@
+(ns ttt.board-spec
+  (:require [speclj.core :refer :all]
+            [ttt.board :refer :all]))
+
+(describe "Board"
+          (it "creates a new empty board"
```

Comment:
formatting seems a bit weird for these. Should it be just 2 spaces?

Reply:
Hm, it's a weird thing vscode is doing. Whenever i reduce the spacing, when I press enter, it auto formats again to use that spacing. I'll dig in to see why it does that only in the tests.